app/templates/frameworks/_framework_actions.html

```diff
@@ -35,8 +35,7 @@
     </a>
   {% endif %}
   <p class="browse-list-item-body">
-    Agree to the terms of the bid, provide supplier information and
-    confirm&nbsp;eligibility.
+    Confirm eligibility, agree to the application terms and provide supplier information.
```

Comment:
I'm guessing the `nbsp` used to be there to ensure there wasn't a word on its own line. Might be worth putting one in the new copy just in case?